pkg/virt-launcher/virtwrap/access-credentials/access_credentials_test.go

```diff
@@ -343,12 +340,13 @@ var _ = Describe("AccessCredentials", func() {
 		})
 
 		matched := false
-		mockConn.EXPECT().QemuAgentCommand(cmdSetPassword, domName).MinTimes(1).DoAndReturn(func(funcCmd string, funcDomName string) (string, error) {
-			if funcCmd == cmdSetPassword {
+		mockDomain.EXPECT().SetUserPassword(user, password, gomock.Any()).MinTimes(1).DoAndReturn(
+			func(u, p string, flags libvirt.DomainSetUserPasswordFlags) error {
+				Expect(u).To(Equal(user))
```

Comment:
Should flags be asserted too? In other calls too, instead of gomock.Any, they should always be zero?

Reply:
Yep let's check it.